STYLEGUIDE.md

```diff
@@ -341,3 +364,65 @@ Next.js is a platform with two different implementations grouped underneath it.
 ![App Router and Pages Router are sitting alongside other very different methods of retrieving data.](/public/images/styleguide/clerk_crowded-tabs.png)
 
 Next.js's two implementations are given equal weight to the other members of the tab bar. Someone unfamiliar with Next.js may become confused, and the ever growing tab bar is harder to navigate at smaller sizes.
+
+### 4.7 Wrap component references in `< />`
+
+Component references should be wrapped in `< />`
+
+#### Do
+
+> Use the `<SignIn />` component.
+
+#### Don't 
+
+> Use the `<SignIn/>` component.
+
+> Use the "SignIn component".
+
+> Use the `SignIn` component.
+
+### 4.8 Specify syntax and filename for terminal commands
+
+If the code should run in a terminal, set the code block's syntax highlighting and filename with `sh filename="terminal"`.
+
+#### Do
+
+```
+​```sh filename="terminal"
+npm i @clerk/nextjs
+​```
+```
+
+#### Don't
+
+```
+​```
+npm i @clerk/nextjs
+​```
+```
+
+### 4.9 Pass properties to components; Parameters to functions
+
+Be sure to use the correct term with components vs functions. 
+
+> Be sure to use the appropriate terms on reference page headings.
+
+#### Do
+
+> Pass the `signUpProps` property to `<SignUp />`. `buildUrlWithAuth()` accepts a `string` for the `to` parameter.
+
+#### Don't
+
+> The `<SignUp />` component accepts the `signUpProps` parameter. The `buildUrlWithAuth()` function accepts the `to` property.
+
+### 4.10 Start and end every Clerk Dashboard operation with specific links to the Clerk Dashboard
+
+When telling learners to perform an operation in the Clerk Dashboard, begin with "In your Clerk Dashboard" and a link to **`https://dashboard.clerk.com/`**. Then end with a link to the page you are directing them to with this URL syntax: **`https://dashboard.clerk.com/last-active?path=PAGE`**
+
+#### Do
+
+> In your [Clerk Dashboard](https://dashboard.clerk.com/), navigate to **[User & Authentication](https://dashboard.clerk.com/last-active?path=user-authentication)**.
+
+#### Don't
+
+> Go to **User & Authentication** in your dasboard.
```

Comment:
This is not the current practice and not what we talked about when we decided on how to write navigation-related copy. I released a very large, time-consuming PR to update all navigation-related copy instances in the docs, and I thought that was the practice we had decided on.
The PR: https://github.com/clerk/clerk-docs/pull/457

Reply:
Oh? This is supposed to canonize that work. What is the difference? The use of "navigate to" instead of "go to?" Update this to reflect your PR then!

"...in the Clerk Dashboard, go to the **[Account Portal](https://dashboard.clerk.com/last-active?path=account-portal)** page and open the **Redirects** tab."